packages/contracts/src/core/permission/PermissionManager.sol

```diff
@@ -218,7 +223,22 @@ abstract contract PermissionManager is Initializable {
     /// @param _who The address (EOA or contract) owning the permission.
     /// @param _permissionId The permission identifier.
     function _grant(address _where, address _who, bytes32 _permissionId) internal virtual {
-        _grantWithCondition(_where, _who, _permissionId, IPermissionCondition(ALLOW_FLAG));
+        if (_where == ANY_ADDR || _who == ANY_ADDR) {
+            revert AnyAddressDisallowedForWhoAndWhere();
+        }
+
+        bytes32 permHash = permissionHash(_where, _who, _permissionId);
+
+        address currentFlag = permissionsHashed[permHash];
+
+        // Means permHash is not currently set.
+        if (currentFlag == UNSET_FLAG) {
+            permissionsHashed[permHash] = ALLOW_FLAG;
+
+            emit Granted(_permissionId, msg.sender, _where, _who, IPermissionCondition(ALLOW_FLAG));
+        } else {
+            // TODO https://aragonassociation.atlassian.net/browse/OS-159
```

Comment:
Should we just remove this `else` branch ?
See https://aragonassociation.atlassian.net/browse/OS-159 @Rekard0

Reply:
Removed in 24b17d9dcc439beeb8ef15249636cb5a52983fd